cmd/roles/roles.go

```diff
@@ -349,54 +353,40 @@ func (mr *MilvusRoles) Run() {
 		queryNode = mr.runQueryNode(ctx, local, alias)
 	}
 
-	if mr.EnableDataCoord {
-		dataCoord = mr.runDataCoord(ctx, local)
-	}
-
 	if mr.EnableDataNode {
 		dataNode = mr.runDataNode(ctx, local, alias)
 	}
 
-	if mr.EnableIndexCoord {
-		indexCoord = mr.runIndexCoord(ctx, local)
-	}
-
 	if mr.EnableIndexNode {
 		indexNode = mr.runIndexNode(ctx, local, alias)
 	}
 
+	if mr.EnableProxy {
+		pctx := log.WithModule(ctx, "Proxy")
+		proxy = mr.runProxy(pctx, local, alias)
+	}
+
 	metrics.Register(Registry)
 
 	<-mr.closed
 
-	var wg sync.WaitGroup
 	// stop coordinators first
 	//	var component
-	coordinators := []component{rootCoord, queryCoord, dataCoord, indexCoord}
+	coordinators := []component{rootCoord, dataCoord, indexCoord, queryCoord}
 	for _, coord := range coordinators {
 		if coord != nil {
-			wg.Add(1)
-			go func(coord component) {
-				defer wg.Done()
-				coord.Stop()
-			}(coord)
+			coord.Stop()
```

Comment:
Do we need to wait standby become to active if enableActiveStandby is true,  or all coord might be unavailable in a short time?